### DISCUSSION TITLE
[Seeking Clarification] Patch OS Enabled : False - Does this affect manual SOUP updates?

### DISCUSSION THREAD

**Discussion by DoughnutPeach** [Security-Onion-Solutions/securityonion#12767] [2.4] [answered]
### Version

2.4.50

### Installation Method

Security Onion ISO image

### Description

upgrading

### Installation Type

Distributed

### Location

cloud

### Hardware Specs

Exceeds minimum requirements

### CPU

32

### RAM

16

### Storage for /

500

### Storage for /nsm

500

### Network Traffic Collection

tap

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

I run a distributed setup and would like to stop automatic OS updates. We will still want to run OS updates on an ad-hoc basis.

I understand that I can do so from the Configuration page as so (https://docs.securityonion.net/en/2.4/soup.html#os-updates).

By configuring `Patch OS Enabled : False`, will SOUP pull OS updates as well?

Thanks for your guidance.

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by TOoSmOotH** [comment, marked as answer]
soup will ask you if you want to update OS packages if it detects you are running out of date packages. If you say yes to update it will update all the OS packages on all grid members.
